configuration.nix

```diff
@@ -1,10 +1,14 @@
-{ config, lib, pkgs, ... }:
 {
+  config,
```

Comment:
Would be nice to do reformatting in a different PR to reduce the line noise.

Reply:
This was done (see first line of the PR description), but the [other PR](https://github.com/nix-community/nixos-generators/pull/239) wasn't merged yet.